code/qcommon/q_shared.c

```diff
@@ -953,21 +953,27 @@ int Q_PrintStrlen( const char *string ) {
 }
 
 
-char *Q_CleanStr( char *string ) {
-	char*	d;
-	char*	s;
-	int		c;
+char *Q_CleanStr(char *string) {
+
+	char  *d;
+	char  *s;
+	int   c;
 
 	s = string;
 	d = string;
-	while ((c = *s) != 0 ) {
-		if ( Q_IsColorString( s ) ) {
+
+	while ((c = *s) != 0) {
```

Comment:
In this way you essentially removed the fix, which is all about this line: 

```c
if ((*s == '^') && (*(s + 1) == '^')) {
    s++;
}
```

If you meant to replace double increment with `s += 2`, sure!